core/src/main/java/nu/yona/server/messaging/service/MessageService.java

```diff
@@ -248,6 +253,19 @@ public void sendMessage(Message message, MessageDestination destinationEntity)
 	}
 
 	@Transactional
+	public void broadcastSystemAdminMessage(String messageText)
+	{
+		userAnonymizedService.getAllUsersAnonymized()
+				.forEach(userAnonymized -> sendSystemAdminMessage(messageText, userAnonymized));
```

Comment:
Another option is to split it up in separate transactions for now using the `TransactionHelper`. Do you think that is a better solution for now?